Bookish/Bookish.Web/Views/Home/AddBook.cshtml

```diff
@@ -30,6 +30,12 @@
             <p class="control-label">ISBN</p>
             @Html.EditorFor(model => model.ISBN, new { htmlAttributes = new { @class = "form-control", required="required" } })
         </div>
+        <div class="form-group">
+            <p class="control-label">Author Name</p>
+            @Html.DropDownListFor(model => model.AuthorID, Model.AuthorNameList, "Please select: ")
+            @*@Html.DropDownList("AuthorList", null, new { htmlAttributes = new { @class = "form-control", required="required" }})*@
```

Comment:
You can probably delete line 36, right? (I'm guessing these were two different versions you were trying but correct me if you meant to have both!)